vector/src/test/java/im/vector/app/features/settings/devices/v2/othersessions/OtherSessionsViewModelTest.kt

```diff
@@ -66,6 +67,7 @@ class OtherSessionsViewModelTest {
     private val fakeRefreshDevicesUseCase = mockk<RefreshDevicesUseCase>(relaxed = true)
     private val fakeSignoutSessionsUseCase = FakeSignoutSessionsUseCase()
     private val fakePendingAuthHandler = FakePendingAuthHandler()
+    private val fakeSharedPreferences = FakeSharedPreferences()
 
     private fun createViewModel(viewState: OtherSessionsViewState = OtherSessionsViewState(defaultArgs)) =
```

Comment:
We should add units tests to cover the new action and the change of the viewState in the initialization.

Reply:
Done.

Reply:
I think tests have not been added for this `ViewModel`.